src/common/Page.ts

```diff
@@ -470,7 +488,7 @@ export class Page extends EventEmitter {
     );
     await page.#initialize();
     if (defaultViewport) {
-      await page.setViewport(defaultViewport);
+      await page.setViewport(defaultViewport).catch(debugErrorIfTargetClosed);
```

Comment:
Can we use try and catch blocks?

Reply:
sure, but in my opinion, it's less readable in this case. Any reasons to prefer one over the other?

Reply:
It's less about readability and more about semantics. Using a `.catch` usually means even if it errors, return something else (which is void in this case). With try-catch, the semantics is try setting the viewport, but if it breaks, let me check out the error.